### ISSUE TITLE
Why can't I set the text in a text field

### ISSUE THREAD

**Issue by ivanceras** [jtomschroeder/cedar#10]
I was playing with this for a bit, but there is no function for changing the text for the text field. I am hoping I could contribute on this project as much as I can.

**Comment by jtomschroeder** [comment]
You're correct. There is currently no method for modifying the contents of text field. I think that'd be a nice addition. Feel free to submit a PR!
